printer/opts.go

```diff
@@ -195,3 +200,47 @@ func WithUpgradeNotice(owner, repo string, opts ...upgrade.Options) *EnableUpgra
 func (c *EnableUpgradeNotice) ApplyToContainerOption(cfg *ContainerOptions) {
 	cfg.UpgradeNotice = upgrade.NewGitHubDetector(c.owner, c.repo, c.upgradeOpts...)
 }
+
+// WithPrettyStyleFromEnv Load a custom style from environment variable
+func WithPrettyStyleFromEnv(envVariable string) (CustomPrettyStyle, error) {
+	path := os.Getenv(envVariable)
+	options, err := parseConfigFile(path)
+
+	return options, err
+}
+
+// WithPrettyStyleFile Load a custom style from file
+func WithPrettyStyleFile(path string) (CustomPrettyStyle, error) {
+	options, err := parseConfigFile(path)
+
+	return options, err
+}
+
+func parseConfigFile(fileName string) (CustomPrettyStyle, error) {
+	var options CustomPrettyStyle
+	styleConfig := style.DefaultConfig(PrettyLayoutGoTpl)
+	if fileName == "" {
+		return options, nil
+	}
+
+	body, err := os.ReadFile(fileName)
```

Comment:
```suggestion
	fileName = filepath.Clean(fileName)
	body, err := os.ReadFile(fileName)
```
let's normalize the filepath before using it.